package.json

```diff
@@ -52,7 +52,7 @@
     "speakingurl": "^14.0.1"
   },
   "optionalDependencies": {
-    "@napi-rs/pinyin": "^1.7.1"
+    "pinyin-pro": "^3.14.0"
```

Comment:
Given `pinyin-pro` is not a native module, we could promote it to standard `dependencies` and remove the try/catch block around its `require` statement.